instrumentation/influxdb-2.4/javaagent/build.gradle.kts

```diff
@@ -17,6 +17,8 @@ dependencies {
   compileOnly("com.google.auto.value:auto-value-annotations")
   annotationProcessor("com.google.auto.value:auto-value")
 
+  testInstrumentation(project(":instrumentation:okhttp:okhttp-3.0:javaagent"))
```

Comment:
Maybe is it  better to describe the context leak problem by adding necessary comment here?

Reply:
It is already described further down where the instrumentation is disabled.

Reply:
> It is already described further down where the instrumentation is disabled.

Yes, I noticed it before, but I was worried that someone might not have noticed the bottom comments.😅